swimlane/core/adapters/record.py

```diff
@@ -402,18 +407,18 @@ def bulk_delete(self, *filters_or_records):
         return self._swimlane.request('DELETE', "app/{0}/record/batch".format(self._app.id), json=data_dict).text
 
 
-def validate_filters_or_records(filters_or_records):
+def validate_filters_or_records_or_ids(filters_or_records_or_ids):
     """Validation for filters_or_records variable from bulk_modify and bulk_delete"""
     # If filters_or_records is empty, fail
-    if not filters_or_records:
-        raise ValueError('Must provide at least one filter tuples or Records')
-    # If filters_or_records is not list of Record or tuple, fail
-    if not isinstance(filters_or_records[0], (Record, tuple)):
-        raise ValueError('Cannot provide both filter tuples and Records')
-    # If filters_or_records is not list of either Record or only tuple, fail
-    _type = type(filters_or_records[0])
-    for item in filters_or_records:
+    if not filters_or_records_or_ids:
+        raise ValueError('Must provide at least one filter tuples, Records, or list of Ids')
```

Comment:
Looks like this change broke a couple of tests, can you take a look?

![image](https://user-images.githubusercontent.com/132009/214163860-8dbfb030-c4cc-4b4e-b145-28a718517231.png)